admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -74,8 +74,24 @@ $(document).ready(function() {
     $('[data-toggle="tooltip"]').tooltip('hide');
     $('[data-toggle="popover"]').popover('hide');
   });
+
+  $('.summary-description-container a[data-toggle="tab"]').on('shown.bs.tab', resetEditor);
 });
 
+/**
+ * Reset active tinyMce editor (triggered when switch language, or switching tabs)
+ */
+function resetEditor() {
+  const languageEditorsSelector = '.summary-description-container .panel.active div.translations.tabbable > div > div.translation-field.active textarea.autoload_rte';
+  $(languageEditorsSelector).each(function(index, textarea) {
+    var editor = tinyMCE.get(textarea.id);
+    if (editor) {
```

Comment:
did you try https://www.tiny.cloud/docs/api/tinymce/tinymce.editor/#nodechanged ?

Reply:
I didn't know it, I just tried but it doesn't work unfortunately